src/Data/Time/PreciseDuration/Internal.purs

```diff
@@ -31,37 +31,37 @@ instance showPreciseDuration :: Show PreciseDuration where
   show (Days d) = "(Days " <> show d <> ")"
   show (Weeks d) = "(Weeks " <> show d <> ")"
 
+unwrapPreciseDuration :: PreciseDuration -> Decimal
+unwrapPreciseDuration = case _ of
```

Comment:
I think conventionally this would just be called `unPreciseDuration`.

Reply:
This is `unwrapPreciseDuration` because it simply projects the number from whatever `PreciseDuration` component you give it. `unPreciseDuration` (in Data.Time.PreciseDuration.purs) will normalise the PreciseDuration to Nanoseconds before projection.